llvm/docs/RISCVUsage.rst

```diff
@@ -228,6 +228,9 @@ The primary goal of experimental support is to assist in the process of ratifica
 ``experimental-zacas``
   LLVM implements the `1.0-rc1 draft specification <https://github.com/riscv/riscv-zacas/releases/tag/v1.0-rc1>`_.
 
+``experimental-zalasr``
+  LLVM implements the `most recent specification <https://github.com/mehnadnerd/riscv-zalasr>`_.
```

Comment:
Can we nail this down to a version or a date? "most recent" is wrong if that repo changes.